src/lingrex/trimming.py

```diff
@@ -131,12 +135,17 @@ def trimmed(self,
 
         skeleton = list(enumerate(self.soundclasses))
         idxs = {i for i, c in skeleton if c in exclude}  # Exclude markers.
-        for idx in candidates:
-            current_skeleton = [c for i, c in skeleton if i not in idxs | {idx}]
-            if any(subsequence_of(s, current_skeleton) for s in skeletons):
-                # Trimming this site leaves a "big enough" remainder.
-                idxs.add(idx)
-            else:
+        for idxss in candidates:
+            if not isinstance(idxss, list):
+                idxss = [idxss]
+            trimmed = False
+            for idx in idxss:
+                current_skeleton = [c for i, c in skeleton if i not in idxs | {idx}]
+                if any(subsequence_of(s, current_skeleton) for s in skeletons):
+                    # Trimming this site leaves a "big enough" remainder.
+                    idxs.add(idx)
+                    trimmed = True
+            if not trimmed:
```

Comment:
@LinguList ... and break if no trimming for any site in a group was possible.

Note that this means no change for the `'core-oriented'` strategy.